package.json

```diff
@@ -3,8 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "materialize-css": "^1.0.0",
```

Comment:
you added materialize and then react-materialize and then you still copied links in index.html